lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -517,7 +517,7 @@ public void shouldValidateCustomClaimOfTypeBoolean() {
     @Test
     public void shouldValidateCustomClaimOfTypeDate() {
         String token = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoxNDc4ODkxNTIxfQ.mhioumeok8fghQEhTKF3QtQAksSvZ_9wIhJmgZLhJ6c";
-        Date date = new Date(1478891521000L);
+        Date date = new Date(1478891521123L);
```

Comment:
Changing the date time to include milliseconds would cause this test to fail without the change to truncate milliseconds.